vignettes/in-packages.Rmd

```diff
@@ -182,17 +188,17 @@ my_function_inside_a_package <- function(...)
 
   if (tidyr_new_interface()) {
     # Freshly written code for v1.0.0
-    out <- tidyr::nest_legacy(df, x, y, z)
+    out <- tidyr::nest(df, data = one_of("x", "y", "z"))
```

Comment:
I have two reasons for changing this:
1. We haven't mentioned `nest_legacy()` yet, so it's a bit out of the blue.
1. Seems like we should show real code, not code that we describe as a "quick and dirty fix".